builtin/logical/database/dbplugin/plugin.go

```diff
@@ -29,6 +29,13 @@ type Statements struct {
 	RenewStatements      string `json:"renew_statements" mapstructure:"renew_statements" structs:"renew_statements"`
 }
 
+// UsernameConfig is used to configure prefixes for the username to be
+// generated.
+type UsernameConfig struct {
```

Comment:
Display names are tied to tokens- the vault user who created the cred. Role name is the name of the database role that created the cred